vllm/config.py

```diff
@@ -3768,7 +3779,8 @@ class VllmConfig:
     lora_config: Optional[LoRAConfig] = None
     speculative_config: SpeculativeConfig = field(default=None,
                                                   init=True)  # type: ignore
-    decoding_config: Optional[DecodingConfig] = None
+    structured_output_config: StructuredOutputConfig = field(
+        default_factory=StructuredOutputConfig, init=True)
```

Comment:
`init` is already `True`

```suggestion
        default_factory=StructuredOutputConfig)
```